<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testcases.TC002_LoginForNegative" tests="4" time="57.899" ignored="3" failures="0" timestamp="24 Mar 2018 04:55:50 GMT" skipped="2" hostname="Vinay" errors="2">
  <testcase name="loginAndLogout" time="24.601" classname="testcases.TC002_LoginForNegative">
    <error type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.type(SeMethods.java:120)
	at pages.LoginPage.enterUserName(LoginPage.java:28)
	at testcases.TC002_LoginForNegative.loginAndLogout(TC002_LoginForNegative.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:744)
]]>
    </error>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="19.219" classname="testcases.TC002_LoginForNegative">
    <skipped/>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="13.017" classname="testcases.TC002_LoginForNegative">
    <skipped/>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="1.062" classname="testcases.TC002_LoginForNegative">
    <error type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.type(SeMethods.java:120)
	at pages.LoginPage.enterUserName(LoginPage.java:28)
	at testcases.TC002_LoginForNegative.loginAndLogout(TC002_LoginForNegative.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:744)
]]>
    </error>
  </testcase> <!-- loginAndLogout -->
  <testcase name="afterTest" classname="wdMethods.ProjectMethods">
    <skipped/>
  </testcase> <!-- afterTest -->
  <testcase name="beforeSuite" classname="wdMethods.ProjectMethods">
    <skipped/>
  </testcase> <!-- beforeSuite -->
  <testcase name="afterSuite" classname="wdMethods.ProjectMethods">
    <skipped/>
  </testcase> <!-- afterSuite -->
</testsuite> <!-- testcases.TC002_LoginForNegative -->
